### DISCUSSION TITLE
Github import option - Enterprise

### DISCUSSION THREAD

**Discussion by ghost** [community/community#66766] [Enterprise]
GitHub Import option

1. Running enterprise version 3.8.4 and I don't see the import repository option on the right top

![image](https://github.com/community/community/assets/20497926/98b0b6f4-3aa1-4d65-8ccf-c02c0a1d0d57)

2. If I want to import from GitLab to GitHub, what are the best options available today?

**Comment by theintrigued** [comment]
For question 2, If I want to import from GitLab to GitHub, what are the best options available today?:

# Here are the possible options:

- GitHub Importer Tool (GitHub) : GitHub provides an official importer tool that allows you to easily migrate your repositories from GitLab to GitHub

- Git Command Line : You can also use Git commands to manually clone your GitLab repository and push it to GitHub

- Third-Party Tools :  There are third-party tools like gitsync and gitlab-mirror that can automate the process of syncing repositories between GitLab and GitHub
 
 # Things to note:
- Check the latest documentation from GitHub and GitLab for migration instructions and ensure you have the required permissions for both repositories.

- Be aware that migrations can impact project settings, permissions, and integrations, so it's wise to test on a non-production repository or create backups to prevent data loss.

- Prioritize careful planning and preparation to ensure a successful repository migration from GitLab to GitHub.
 
hope this helps!

**Reply by ghost** [reply]
Where can I access the GitHub Importer tool?

**Reply by theintrigued** [reply]
Here for starters:
https://docs.github.com/en/migrations/importing-source-code/using-github-importer/about-github-importer

and then:
https://docs.github.com/en/migrations/importing-source-code/using-github-importer/importing-a-repository-with-github-importer

finally:
https://docs.github.com/en/migrations/importing-source-code/using-github-importer
